dnf/cli/commands/remove.py

```diff
@@ -42,7 +42,7 @@ class RemoveCommand(commands.Command):
                    'erase-na': hawkey.FORM_NA,
                    'erase-nevra': hawkey.FORM_NEVRA}
 
-    aliases = ('remove', 'erase') + tuple(nevra_forms.keys())
+    aliases = ('remove', 'erase', 'r', 'e', 'rm') + tuple(nevra_forms.keys())
```

Comment:
I don't like the idea of one letter short forms. That's too easily confusing, especially with package names like `e` out in the wild. (Yes, it's dumb, but they exist and I deal with them).